src/main/java/com/minecolonies/entity/pathfinding/AbstractPathJob.java

```diff
@@ -775,12 +787,13 @@ protected int getGroundHeight(Node parent, @NotNull BlockPos pos)
 
         //  Do we have something to stand on in the target space?
         final IBlockState below = world.getBlockState(pos.down());
-        if (isWalkableSurface(below) == SurfaceType.WALKABLE)
+        SurfaceType walkability = isWalkableSurface(below);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'walkability' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)